mappings/net/minecraft/util/JavaOps.mapping

```diff
@@ -0,0 +1,103 @@
+		ARG 1 map
+	METHOD createMap createMap (Ljava/util/stream/Stream;)Ljava/lang/Object;
+		ARG 1 stream
+	METHOD createNumeric createNumeric (Ljava/lang/Number;)Ljava/lang/Object;
+		ARG 1 n
+	METHOD createShort createShort (S)Ljava/lang/Object;
+		ARG 1 s
+	METHOD createString createString (Ljava/lang/String;)Ljava/lang/Object;
+		ARG 1 string
+	METHOD empty empty ()Ljava/lang/Object;
+	METHOD emptyList emptyList ()Ljava/lang/Object;
+	METHOD emptyMap emptyMap ()Ljava/lang/Object;
+	METHOD getBooleanValue getBooleanValue (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getByteBuffer getByteBuffer (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getIntStream getIntStream (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getList getList (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getLongStream getLongStream (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getMap getMap (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getMapEntries getMapEntries (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getMapValues getMapValues (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getNumberValue getNumberValue (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getStream getStream (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD getStringValue getStringValue (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+	METHOD m_afyiirgn (Lcom/google/common/collect/ImmutableMap$Builder;Lcom/mojang/datafixers/util/Pair;)V
+		ARG 1 pair
+	METHOD m_boxzzpcu (Ljava/util/Map$Entry;)Lcom/mojang/datafixers/util/Pair;
+		ARG 0 entry
+	METHOD m_mrnxttti entryStream (Ljava/util/Map;)Ljava/util/stream/Stream;
+		ARG 0 map
+	METHOD m_rxpnwroz toMap (Lcom/mojang/serialization/MapLike;)Ljava/util/Map;
+		ARG 0 mapLike
+	METHOD m_sfvskaol (Ljava/lang/Object;)Ljava/lang/Object;
+		ARG 0 object
+	METHOD mapBuilder mapBuilder ()Lcom/mojang/serialization/RecordBuilder;
+	METHOD mergeToList mergeToList (Ljava/lang/Object;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+		ARG 2 o2
+	METHOD mergeToList mergeToList (Ljava/lang/Object;Ljava/util/List;)Lcom/mojang/serialization/DataResult;
+		ARG 1 object
+		ARG 2 list
+	METHOD mergeToMap mergeToMap (Ljava/lang/Object;Lcom/mojang/serialization/MapLike;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+		ARG 2 mapLike
+	METHOD mergeToMap mergeToMap (Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+		ARG 2 o2
+		ARG 3 o3
+	METHOD mergeToMap mergeToMap (Ljava/lang/Object;Ljava/util/Map;)Lcom/mojang/serialization/DataResult;
+		ARG 1 o
+		ARG 2 map
+	METHOD remove remove (Ljava/lang/Object;Ljava/lang/String;)Ljava/lang/Object;
+		ARG 1 map
+		ARG 2 s
```

Comment:
```suggestion
		ARG 2 string
```
`s` is used for short in other places, and I think if single letters are used, they should be for primitives (and maybe `Object`s)